_posts/2017-01-17-forced-git-hub-a-reflection-on-a-new-website-using-git-hub-and-jekyll.md

```diff
@@ -0,0 +1,54 @@
+
+### Success
+
+After about eight months of editing old content and creating new content, every employee in our office has successfully edited our website. Everyone has their notes and instructions on how to be successful and almost no two workflows are completely alike. You can write up [multiple ways]({{ "/about/contributing" | prepend: site.baseurl }}) to [create and edit content]({{ "/about/contributing/edit-existing" | prepend: site.baseurl }}) and it will differ from your coworker. The GitHub flow introduced having a core group of contributors to look over content. Only these people have the access to accept pull requests. This was the first website with a review process and it has been very useful. Gone are the days of no code reviews. The organization and quality of our content has greatly increased.
+
+### Engagement
+
+We have had external community commits but fewer than we were expecting. Users rarely use the [GitHub issues to report problems](https://github.com/agrc/gis.utah.gov/issues/new) with our site but the ones that do [have had great conversations](https://github.com/agrc/gis.utah.gov/issues/248) and results. We suspect if users do not have a GitHub account we do not get the conversion. We also suspect that people do not think their story is worth telling when it really is!
+
+### Green Grass
+
+We are pleased with our website and hope you are too! We all rely on our [notes and cheatsheets](https://github.com/agrc/gis.utah.gov/blob/master/README.md) and our workflows for being successful. GitHub and git are hard. Markdown is difficult. Jekyll is complicated. We thought it would be fun and useful as an office to learn to use GitHub and the other technologies involved in a static website. We are an open source agency with our data and our applications. All of our applications reside on GitHub so having our staff familiar was a welcomed side affect. Some employees have seen the benefits and use GitHub for personal GIS scripts and tools. The forced learning process could have been smoother and if we did it again we would adjust our approach but we made it happen as a team.
```

Comment:
>The forced learning process could have been smoother and if we did it again we would adjust our approach but we made it happen as a team.

The forced learning process could have been smoother. If we did it again we would adjust our approach. But in the end we made it happen as a team.